momento-redis/sorted_set_commands_test.go

```diff
@@ -0,0 +1,355 @@
+package momento_redis_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/momentohq/client-sdk-go/auth"
+	"github.com/momentohq/client-sdk-go/config"
+	"github.com/momentohq/client-sdk-go/momento"
+	momentoredis "github.com/momentohq/momento-go-redis-client/momento-redis"
+
+	. "github.com/momentohq/momento-go-redis-client/momento-redis/test_helpers"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redis/go-redis/v9"
+)
+
+var _ = Describe("Sorted Set methods", func() {
+	sContext := NewSharedContext()
+	BeforeEach(func() {
+		cacheName := "default_cache"
```

Comment:
Could you please change this to use a UUID-based cache name like you did with the scalar tests?